docs/source/glossaries.rst

```diff
@@ -0,0 +1,107 @@
+QP          Queue Pair(Infiniband)                                        
+RC          Reliable Connection (Infiniband)                              
+rcache      Registration Cache                                            
+RDMA        Remote Direct Memory Access                                   
+REQ         Request                                                       
+rkey        Remote KEY                                                    
+RMA         Remote Memory Access                                          
+RNR         Receiver Not Ready                                            
+RoCE        RDMA over Converged Ethernet                                  
+ROCm        Radeon Open Compute platform(AMD)                             
+RTE         Run Time Environment                                          
+RX          Receive                                                       
+Skb         Socket Buffer                                                 
+sm          Shared Memory                                                 
+SM          Subnet Manager(Infiniband)                                    
+SockCM      Socket Connection Manager                                     
+SRQ         Shared Receive Queue                                          
+SYSV        UNIX System V                                                 
+tl          Transport Layer                                               
+TLS         Transpot LayerS                                               
+TM          Tag Matching                                                  
+TX          Transmit                                                      
+UC          Unreliable Connection (Infiniband)                            
+UCC         Unified Collective Communications                             
+UCF         Unified Communication Framework                               
+UCM         Unified Communication Memory                                  
+UCP         Unified Communication Protocols Higher level API              
+UCS         Unified Communication Service Common utilities.               
+UCT         Unified Communication Transport Lower level API               
+UCX         Unified Communication X                                       
+UD          Unreliable Datagram (Infiniband)                              
+uGNI        user level generic network interface(Cray)                    
+UMR         User mode memory registration                                 
+VPI         Virtual Protocol Interconnect                                 
+WIP-DNM     Work-in-Progress-Do-Not-Merge                                 
+WQ          Work Queue(Infiniband)                                        
+WQE         Work Queue Elements (pronounce WOOKIE)                        
+WR          Work Request                                                  
+XPMEM       cross partition memory                                        
```

Comment:
Linux kernel module originally developed by SGI implementing cross process memory mapping with  zero copy (https://github.com/hjelmn/xpmem)

Reply:
I modified this. But, I think it's a little longer description.